NeuroID/src/main/java/com/neuroid/tracker/compose/JetpackCompose.kt

```diff
@@ -82,4 +85,58 @@ class JetpackComposeImpl(
             )
         )
     }
+
+    @Composable
+    override fun trackElement(
+        elementState: String,
+        elementName: String,
+        pageName: String,
+    ) {
+        val clipboardManager = LocalClipboardManager.current
+        // save instance of previous text and previous pasted text hashcode
+        var previousText by remember { mutableStateOf(elementState) }
+        var lastPastedHashCode by remember { mutableStateOf(0) }
+
+        // handle the text changes
+        LaunchedEffect(elementState) {
+            if (!neuroID.isStopped() && elementState != previousText) {
+                val changeText = neuroID.nidComposeTextWatcher.getTextChange(previousText, elementState)
+                val clipboardContent = clipboardManager.getText()?.text ?: ""
+                // check for paste, send paste event if so
+                if (neuroID.nidComposeTextWatcher.isPaste(changeText, clipboardContent, lastPastedHashCode)) {
+                    neuroID.captureEvent(
+                        type = PASTE,
+                        tg =
+                            hashMapOf(
+                                "attr" to getAttrJson(elementState),
+                                "et" to "text",
+                            ),
+                        tgs = "$pageName:$elementName",
+                        v = "S~C~~${clipboardContent.length}",
+                        hv = clipboardContent.getSHA256withSalt().take(8),
+                        attrs =
+                            listOf(
+                                mapOf(
+                                    "clipboardText" to "S~C~~${clipboardContent.length}",
+                                ),
+                            ),
+                    )
+                    lastPastedHashCode = clipboardContent.hashCode()
+                }
+                neuroID.captureEvent(
+                    type = INPUT,
+                    tg =
+                        hashMapOf(
+                            "attr" to getAttrJson(elementState),
+                            "etn" to INPUT,
+                            "et" to "text",
+                        ),
+                    tgs = "$pageName:$elementName",
+                    v = "S~C~~${elementState.length}",
+                    hv = elementState.getSHA256withSalt().take(8),
+                )
+                previousText = elementState
```

Comment:
We could also add the `attrs` field to show that its from a compose element like the button and page events? attrs = listOf(
                sdkMap
            )
where sdkMap =mapOf(
        "jetpackCompose" to true,
    )

Reply:
added. thanks!